modules/overrides/limits.go

```diff
@@ -96,6 +98,9 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&l.PerTenantOverrideConfig, "limits.per-user-override-config", "", "File name of per-user overrides.")
 	_ = l.PerTenantOverridePeriod.Set("10s")
 	f.Var(&l.PerTenantOverridePeriod, "limits.per-user-override-period", "Period with this to reload the overrides.")
+
+	l.MetricsGeneratorForwarderQueueSize = 10
+	l.MetricsGeneratorForwarderWorkers = 1
```

Comment:
Nitpick: this doesn't match with `defaultWorkerCount` defined in `forwarder.go`. I think we can either:
- export `defaultWorkerCount` and use it here instead of defining the same default twice
- don't set the defaults here, since we added the logic that checks for `workerCount == 0` I don't think this is necessary anymore

Reply:
Good catch. This was something left before adding the defaults. Went with option #2.